### ISSUE TITLE
Bug: Variants without ClinVar calls are getting duplicated output

### ISSUE THREAD

**Issue by jungkim2** [diskin-lab-chop/AutoGVP#85]
#### Provide the command used or report the bug here
Variants without ClinVar calls are getting duplicated output

```

vcf_id	Ref.Gene	Func.refGene	ExonicFunc.refGene	AAChange.refGene	Stars	Intervar_evidence	intervar_adjusted_call	ID	final_call	Reasoning_for_call
1-64834468-CTTCT-C	JAK1	UTR3	.	.	NA	InterVar: Uncertain significance PVS1=0 PS=[0, 0, 0, 0, 0] PM=[0, 1, 0, 0, 0, 0, 0] PP=[0, 0, 0, 0, 0, 0] BA1=0 BS=[0, 0, 0, 0, 0] BP=[0, 0, 0, 0, 0, 0, 0, 0]	No	NA	Uncertain_significance	Intervar
1-64834468-CTTCT-C	JAK1	UTR3	.	.	NA	InterVar: Uncertain significance PVS1=0 PS=[0, 0, 0, 0, 0] PM=[0, 1, 0, 0, 0, 0, 0] PP=[0, 0, 0, 0, 0, 0] BA1=0 BS=[0, 0, 0, 0, 0] BP=[0, 0, 0, 0, 0, 0, 0, 0]	Not adjusted, clinVar	NA	Uncertain_significance	ClinVar
``` 

#### Add Session info
Run `sessionInfo()` and post the output below

**Comment by jungkim2** [comment]
Maybe for more logical sense would below might work better? 

1. If ClinVar calls with 1, 1NR, 2, 3, 4 stars take ClinVar call 
2. If ClinVar 0 star, move down the pipeline of InterVar
3. If InterVar has autoPVS1, PP5, BP6 do InterVar adjustment
4. If none of the above take InterVar as is

**Comment by rjcorb** [comment]
resolved with PR #94
